tests/classes/event_properties/biosero/test_automation_system_name.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from lighthouse.classes.event_properties.definitions import RunID
+from lighthouse.classes.event_properties.definitions.biosero import AutomationSystemName, RunInfo
+from lighthouse.constants.fields import (
+    FIELD_CHERRYTRACK_LIQUID_HANDLER_SERIAL_NUMBER,
+    FIELD_EVENT_RUN_ID,
+    FIELD_CHERRYTRACK_USER_ID,
+    FIELD_CHERRYTRACK_AUTOMATION_SYSTEM_MANUFACTURER,
+    FIELD_CHERRYTRACK_AUTOMATION_SYSTEM_NAME,
+)
+from http import HTTPStatus
+
+
+def test_robot_serial_number_valid(app):
+    assert AutomationSystemName(RunInfo(RunID({FIELD_EVENT_RUN_ID: 1}))).valid() is True
+
+
+@pytest.mark.parametrize("run_id", [5])
+def test_robot_serial_number_value_successful(app, run_id, mocked_responses, cherrytrack_mock_run_info):
+    with app.app_context():
+
+        expected_response = {
+            "data": {
+                "id": run_id,
+                FIELD_CHERRYTRACK_USER_ID: "user1",
+                FIELD_CHERRYTRACK_LIQUID_HANDLER_SERIAL_NUMBER: "aLiquidHandlerSerialNumber",
+                FIELD_CHERRYTRACK_AUTOMATION_SYSTEM_MANUFACTURER: "biosero",
+                FIELD_CHERRYTRACK_AUTOMATION_SYSTEM_NAME: "CPA",
+            }
+        }
+
+        val = AutomationSystemName(RunInfo(RunID({FIELD_EVENT_RUN_ID: run_id}))).value
+
+        assert val == expected_response["data"][FIELD_CHERRYTRACK_AUTOMATION_SYSTEM_NAME]
+
+
+@pytest.mark.parametrize("run_id", [5])
```

Comment:
I see these are stacked but they are single parameters so could these also be combined since you are not needed the combinations of these parameters like in the first and third example of https://docs.pytest.org/en/6.2.x/parametrize.html.

Reply:
Do you mean?
```
@pytest.mark.parametrize("run_id, cherrytrack_mock_run_info_status, cherrytrack_run_info_response", [5, HTTPStatus.INTERNAL_SERVER_ERROR, {.......}])
```

Reply:
I'm thinking that way is not so easy to read the value of each param